ui/src/components/DialogAction/index.jsx

```diff
@@ -95,10 +95,7 @@ function DialogAction(props) {
         {body && <DialogContentText component="div">{body}</DialogContentText>}
       </DialogContent>
       <DialogActions>
-        <Button
-          disabled={actionExecuting}
-          onClick={onClose}
-          action={actions => actions && actions.focusVisible()}>
```

Comment:
This behavior (`fucusVisible()`) makes typing the comment impossible. Seems not a big problem remove it. Let me know if has any other approach.

Reply:
I am not worried about it but there might be a reason it the code is calling the focus visible on that button. So I will take a closer look

Reply:
@jcristau this was focusing / highlighting the cancel button on the dialogs (i.e. the signing-off pop-ups) This was probably done so pressing enter would cancel the action / close the pop-up.

I think this change an improvement. Now we can tab through the buttons in the dialog and press enter to "click" on them, which is the way they should behave. Also the button wont take away the focus from other UI elements we might want to put in the dialogs (e.g. the text box on this PR)

Reply:
@jcristau this code was focusing / highlighting the cancel button on the dialogs (i.e. the signing-off pop-ups) This was probably done so pressing enter would cancel the action / close the pop-up.

I think this change is an improvement. Now we can tab through the buttons in the dialogs and press enter to "click" them (this is the way they should behave imo). Also the button wont take away the focus from other UI elements we might want to put in these dialogs (e.g. @allan-silva's text box)

Reply:
@jcristau this code was focusing / highlighting the cancel button on the dialogs (i.e. the signing-off pop-ups) This was probably done so pressing enter would cancel the action / close the pop-up.

I think this change is an improvement. Now we can tab through the buttons in the dialogs and press enter to "click" them (this is the way they should behave imo). Also the button wont take away the focus from other UI elements we might want to put in the body of these dialogs (e.g. @allan-silva's TextField)

Reply:
@gabrielBusta Thanks for take a close look on this!